src/dotnet/commands/dotnet-run3/Run3Command.cs

```diff
@@ -80,21 +84,16 @@ private ICommand GetRunCommand()
                 globalProperties.Add("TargetFramework", Framework);
             }
 
-            ProjectInstance projectInstance = new ProjectInstance(Project, globalProperties, null);
+            Project project = new Project(Project, globalProperties, null);
 
-            string runProgram = projectInstance.GetPropertyValue("RunCommand");
+            string runProgram = project.GetPropertyValue("RunCommand");
```

Comment:
Will this work if the project hasn't been built yet? If not, should we have an error message for when the project is not built and the nobuild flag is passed?